examples/map_transformations/downloading_hmi.py

```diff
@@ -1,9 +1,10 @@
 """
-===========================================
-Downloading and plotting an HMI magnetogram
-===========================================
+==============================================
+Rotating HMI maps so they're not 'upside-down'
```

Comment:
```suggestion
Solar physicists hate it, One Simple Trick to rotate your upside-down HMI image
```
I think this needs to be more Buzzfeed style

Reply:
> They opened a HMI FITS file and you won't believe what happened next